packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -262,13 +262,15 @@ class AppSizeController {
     _diffCallGraphRoot.value = null;
     _oldDiffCallGraph = null;
     _newDiffCallGraph = null;
+    _isDeferredApp.value = false;
```

Comment:
Let's split this into a separate PR so we can add a test for it at the same time